platform/android/scripts/generate-style-code.mjs

```diff
@@ -0,0 +1,470 @@
+import fs from "node:fs";
+import ejs from "ejs";
+import spec from "./android-style-spec-overrides.mjs";
+import _ from "lodash";
+
+// FIXME: https://github.com/mapbox/mapbox-gl-native/issues/15008
+//delete spec.layout_circle["circle-sort-key"]
+//delete spec.layout_line["line-sort-key"]
+//delete spec.layout_fill["fill-sort-key"]
+
+import { writeIfModified, camelize, snakeCaseUpper, iff, camelizeWithLeadingLowercase } from "../../../scripts/style-code.mjs";
+
```

Comment:
Since this script generated code I would consider adding a comment about how it works, what it does and how.
Also consider adding more docs to the methods in this file. 
Try and think about a situation where someone needs to understand what this files should be doing and you are not around to explain...

Reply:
This script merely got moved. I am not an expert on it either.

Reply:
Git thinks it's a new file... Feel free to ignore my comments related to this file, or take this as an opportunity to learn it 😃

Reply:
Yeah I deleted it by accident (forgot to push it after a move).